pkg/cmd/install.go

```diff
@@ -210,6 +200,68 @@ func (o *installCmdOptions) install(_ *cobra.Command, _ []string) error {
 			platform.Spec.Build.Timeout.Duration = d
 		}
 
+		if len(o.repositories) > 0 {
+			o.mavenSettings = fmt.Sprintf("configmap:%s-maven-settings/settings.xml", platform.Name)
+
+			settings := maven.NewSettings()
```

Comment:
Now that we generate the settings file, we may want to handle the local repository the same way.

Reply:
I think the problem is still the same, what if the user does not set it ?

Reply:
Right, it's still need a merge strategy, which can be addressed later.